INSTALL.md

```diff
@@ -108,7 +124,18 @@ $ make
 
 The executable will be located on the `bin` directory.
 
-### Installing
+#### Windows
+
+Run CMake to generate a solution in visual studio (note: modify the command below to reflect your vcpkg installation folder).
+
+```bash
+$ cmake -DCMAKE_TOOLCHAIN_FILE=D:\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake ..
```

Comment:
I am not sure but I found it is more explicit to replace this line by this : 
`cmake -DCMAKE_TOOLCHAIN_FILE=${PATH_TO_VCPKG}\\scripts\\buildsystems\\vcpkg.cmake ..` ?